share/initialization/common

```diff
@@ -177,7 +177,7 @@ elif [ "$(uname)" = Darwin ] 2>/dev/null; then
 fi
 
 # define some basic variables if missing
-: ${PAGER:=less} ${EDITOR:=vi} ${FCEDIT:=$EDITOR}
+: ${PAGER:=less} ${VISUAL:=vi} ${EDITOR:=ed} ${FCEDIT:=$VISUAL}
```

Comment:
I don't think anyone would be happy to use ed as the default editor.

```suggestion
: ${PAGER:=less} ${VISUAL:=vi} ${EDITOR:=vi} ${FCEDIT:=$VISUAL}
```

Reply:
I'm confident this wouldn't affect many people. Most Unix programs (including now Yash itself) read VISUAL before EDITOR. Most people set EDITOR to their preferred program in their profile anyways.

Reply:
`EDITOR` is only relevant when using a tool that does not recognize `VISUAL`. Do people really want to use ed just because the tool only supports `EDITOR`? I don't see the point of preferring ed over vi.